src/components/fabToolbar/fabToolbar.js

```diff
@@ -0,0 +1,134 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('material.components.fabToolbar', [])
+    .directive('mdFabToolbar', MdFabToolbarDirective);
+
+  /**
+   * @ngdoc directive
+   * @name mdFabToolbar
+   * @module material.components.fabToolbar
+   *
+   * @restrict E
+   *
+   * @description
+   *
+   * The `<md-fab-toolbar>` directive is used present a toolbar of elements (usually `<md-button>`s) for quick access to
+   * common actions when a floating action button is activated (via hover or keyboard navigation).
+   *
+   * @usage
+   *
+   * <hljs lang="html">
+   * <md-fab-toolbar>
+   *   <md-fab-trigger>
+   *     <md-button><md-icon icon="/img/icons/plus.svg"></md-icon></md-button>
+   *   </md-fab-trigger>
+   *
+   *   <md-fab-actions>
+   *     <md-button><md-icon icon="/img/icons/user.svg"></md-icon></md-button>
+   *     <md-button><md-icon icon="/img/icons/group.svg"></md-icon></md-button>
+   *   </md-fab-actions>
+   * </md-fab-toolbar>
+   * </hljs>
+   *
+   * @param {expression=} md-open Programmatically control whether or not the toolbar is visible.
+   */
+  function MdFabToolbarDirective($timeout) {
+    return {
+      restrict: 'E',
+      transclude: true,
+      template: [
+        '<div class="md-fab-toolbar-wrapper">',
+        '  <div class="md-fab-toolbar-background"></div>',
+        '  <div ng-transclude></div>',
+        '</div>'
+      ].join('\n'),
+
+      scope: {
+        isOpen: '=?mdOpen',
+      },
+
+      controller: controller,
+
+      link: link
+    };
+
+    function controller($scope, $element) {
+      var vm = this;
+
+      // Set the default to be closed
+      if (typeof $scope.isOpen === 'undefined') {
+        $scope.isOpen = false;
+      }
+
+      vm.focus = function() {
+        // TODO: Why does this need to be inside a $timeout???
+        $timeout(function() {
```

Comment:
Use `$scope.$apply('isOpen = true')` to update the scope variable and trigger databinding updates.